wagtail/core/tests/test_streamfield.py

```diff
@@ -202,27 +215,33 @@ class InvalidStreamModel(models.Model):
         self.assertEqual(errors[0].obj, InvalidStreamModel._meta.get_field("body"))
 
 
+class TestJSONSystemCheck(TestSystemCheck):
+    use_json_field = True
+
+
 class TestStreamValueAccess(TestCase):
+    model = StreamModel
+
     def setUp(self):
-        self.json_body = StreamModel.objects.create(
+        self.json_body = self.model.objects.create(
             body=json.dumps([{"type": "text", "value": "foo"}])
         )
-        self.nonjson_body = StreamModel.objects.create(body="<h1>hello world</h1>")
```

Comment:
This is only used in `test_can_read_non_json_content`. I moved it there so it's easy to be skipped for `JSONField`.